intel_pytorch_extension_py/ops/jit.py

```diff
@@ -17,23 +16,28 @@ def script_(obj, optimize=None, _frames_up=0, _rcb=None):
     if core.get_jit_opt() and hasattr(jit_m, '_c'):
         # Disable mix precision in model fusion, since mixed precision cannot
         # bring any benefits for inference, but will lead to loss of accuracy
-        orig_mixed_type = ipex.get_auto_mix_precision()
-        ipex.enable_auto_mix_precision(None)
-        jit_m = wrap_cpp_module(torch._C._jit_pass_fold_convbn(jit_m._c))
-        ipex.enable_auto_mix_precision(orig_mixed_type)
+        if core.get_mix_bf16_fp32():
+            core.disable_mix_bf16_fp32()
+            jit_m = wrap_cpp_module(torch._C._jit_pass_fold_convbn(jit_m._c))
+            core.enable_mix_bf16_fp32()
+        else:
+            jit_m = wrap_cpp_module(torch._C._jit_pass_fold_convbn(jit_m._c))
     return jit_m
 
 def trace_(func, example_inputs, *args, **kwargs):
     # Disable mix precision. torch.jit.trace will check the traced output
     # against what is expected. Since mix precision will lead to
     # loss of accuracy, this will raise warning during torch.jit.trace
-    orig_mixed_type = ipex.get_auto_mix_precision()
-    ipex.enable_auto_mix_precision(None)
-    jit_m = orig_trace(func, example_inputs, *args, **kwargs)
-
-    if core.get_jit_opt() and hasattr(jit_m, '_c'):
-        jit_m = wrap_cpp_module(torch._C._jit_pass_fold_convbn(jit_m._c))
-    ipex.enable_auto_mix_precision(orig_mixed_type)
+    if core.get_mix_bf16_fp32():
+        core.disable_mix_bf16_fp32()
+        jit_m = orig_trace(func, example_inputs, *args, **kwargs)
+        if core.get_jit_opt() and hasattr(jit_m, '_c'):
+            jit_m = wrap_cpp_module(torch._C._jit_pass_fold_convbn(jit_m._c))
+        core.enable_mix_bf16_fp32()
+    else:
+        jit_m = orig_trace(func, example_inputs, *args, **kwargs)
```

Comment:
a little bit duplication here. we may store the original mix state in a variable.

Reply:
I don't think need a variable to store the original mix state, we will check mix state before tracing and the former case do it.

Reply:
LGTM. Thanks